packages/loader/container-loader/api-report/container-loader.legacy.alpha.api.md

```diff
@@ -48,9 +48,9 @@ export interface IFluidModuleWithDetails {
     module: IFluidModule;
 }
 
-// @alpha (undocumented)
+// @alpha @deprecated (undocumented)
 export interface ILoaderOptions extends ILoaderOptions_2 {
-    // (undocumented)
+    // @deprecated (undocumented)
```

Comment:
What does `undocumented` mean?

Reply:
The property is undocumented in the interface.